Content.Server/Botany/Components/PlantHolderComponent.cs

```diff
@@ -1,90 +1,94 @@
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
 namespace Content.Server.Botany.Components
 {
     [RegisterComponent]
     public sealed partial class PlantHolderComponent : Component
     {
-        [ViewVariables]
+        [DataField("nextUpdate", customTypeSerializer: typeof(TimeOffsetSerializer))]
         public TimeSpan NextUpdate = TimeSpan.Zero;
+        [ViewVariables(VVAccess.ReadWrite), DataField("updateDelay")]
         public TimeSpan UpdateDelay = TimeSpan.FromSeconds(3);
 
-        [ViewVariables]
+        [DataField("lastProduce")]
```

Comment:
What is the rationale for making this (and other similar fields) data fields? `LastProduce` is an internal state variable, and it doesn't seem like it makes a lot of sense that it would be specified in YAML or overriden in mapping.